issue_processing/jira_issue_processing.py

```diff
@@ -432,12 +544,13 @@ def print_to_disk(issues, results_folder):
     csv_writer.write_to_csv(output_file, lines)
 
 
-def print_to_disk_bugs(issues, results_folder, skip_history):
+def print_to_disk_bugs(issues, results_folder):
     """Sorts of bug issues and prints them to file 'bugs-jira.list' in result folder
+    This method prints in a new format which is consistent to the new GitHub format.
```

Comment:
What is "the new GitHub format"? :wink:  Please refer to the corresponding function in the other file.